src/components/shop/shop-layout.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Scaling and positioning for shop UI elements.
+ *
+ * @module components/shop
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+
+export const getSafeArea = layout => layout.getSafeArea({}, false);
+
+export const getXPos = (container, safeArea, padding) => safeArea.width / 2 - container.getBounds().width / 2 - padding;
+
+export const getYPos = (metrics, safeArea) => {
+    const { verticals, verticalBorderPad } = metrics;
+    const padding = (safeArea.y - verticals.top) / 2 + verticalBorderPad / 2;
+    return verticals.top + padding;
+};
+
+export const getScaleFactor = args => {
+    const { metrics, container, fixedWidth, safeArea } = args;
+    container.setScale(1);
```

Comment:
dunno if a get function that sets the scale to 1 might cause confusing behaviour - I know this is used to set it afterwards anyway. is there maybe another way to get the width and height of the container? (can't remember been a while since I last looked at Phaser :D)

Reply:
Maybe I need to use the scale to normalise the rest of the values. It does feel a bit dirty. Will look.

Reply:
this was a good shout- I was able to dispense with the `.setScale` call. b283938